### ISSUE TITLE
An Example of how to use this would be nice

### ISSUE THREAD

**Issue by Pita** [kashif/node-cuda#1]
....

**Comment by nekinie** [comment]
Any news?

**Comment by gubarez** [comment]
check out the test

**Comment by CodersBrothers** [comment]
Maybe its abandoned this module? No there are wiki, no there are examples to process graphics, no there are any docu,.... who develop this? LOOOOL

**Comment by danfinlay** [comment]
This is really too bad, CUDA bindings for node.js would be amazing.

**Comment by ktamiola** [comment]
Indeed! Please provide some examples. **I would be happy to help** with putting together a decent documentation!

**Comment by Erendis42** [comment]
ready.

**Comment by Chris2011** [comment]
Project still alive or not? Last commits from 2014. Any alternatives?

**Comment by Codzart** [comment]
gpu.js seems closest, but nodejs support seems sketchy
